src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -681,6 +702,7 @@ void BaseCouplingScheme::initializeTXTWriters()
     if (not doesFirstStep()) {
       _convergenceWriter->addData("TimeWindow", io::TXTTableWriter::INT);
       _convergenceWriter->addData("Iteration", io::TXTTableWriter::INT);
+      _convergenceWriter->addData("ReachedMinIter", io::TXTTableWriter::INT);
```

Comment:
See above. Additionally, there seems to be quite some code duplication in this part.

Reply:
See https://github.com/precice/precice/pull/1841#discussion_r1372875827